handlers/BillTechButtonInsertHandler.php

```diff
@@ -60,6 +60,25 @@ public function notifyCustomerDataParse(array $hook_data = array())
 		return $hook_data;
 	}
 
+	public function messageaddCustomerDataParse(array $hook_data = array())
+	{
+		$customerid = $hook_data['data']['id'];
+		$link = self::getPaymentLink('balance', $customerid, ['utm_medium' => 'email']);
+		if (isset($hook_data['data']['contenttype'])) {
+			if ($customerid = $hook_data['data']['contenttype'] == 'text/html') {
+				$cashBtnCode = $this->getBtnCode($hook_data['mail_format'], $link);
+				$hook_data['body'] = preg_replace('/%billtech_btn/', $cashBtnCode, $hook_data['body']);
+			} else {
+				$hook_data['body'] = preg_replace('/%billtech_btn/', $this->createEmailButton('txt', $link), $hook_data['body']);
+			}
+		} else {
+			$hook_data['body'] = preg_replace('/%billtech_btn/', $this->createEmailButton('txt', $link), $hook_data['body']);
+		}
```

Comment:
Tego ifa można znormalizować - w obu elsach jest to samo.

Reply:
Zastanawiam się jeszcze czemu w przypadku emaila w formacie html używamy getBtnCode, a nie createEmailButton?